frontend/packages/console-dynamic-plugin-sdk/src/app/components/utils/headings.tsx

```diff
@@ -0,0 +1,208 @@
+import * as React from 'react';
+import { Breadcrumb, BreadcrumbItem, Button, SplitItem, Split } from '@patternfly/react-core';
+import * as classNames from 'classnames';
+import * as _ from 'lodash';
+import { Link } from 'react-router-dom';
+import { ActionsMenu, ResourceIcon } from '@console/internal/components/utils'; // ??! TODO use direct imports instead
+import { Status } from '@console/shared/src/components/status/Status'; // ??! TODO update after the separate PR gets merged
+import { K8sKind } from '../../../api/common-types';
+import {
+  FirehoseResult,
+  KebabOption,
+  K8sResourceKindReference,
+} from '../../../extensions/console-types';
+import { K8sResourceKind } from '../../k8s/actions/k8s';
+import { connectToModel } from '../../kinds';
+import { ResourceStatus } from './resource-status';
+
+export const BreadCrumbs: React.SFC<BreadCrumbsProps> = ({ breadcrumbs }) => (
+  <Breadcrumb>
+    {breadcrumbs.map((crumb, i, { length }) => {
+      const isLast = i === length - 1;
+
+      return (
+        // eslint-disable-next-line react/no-array-index-key
+        <BreadcrumbItem key={i} isActive={isLast}>
```

Comment:
@fhlavac do you know if there a something else in the breadcrumb map we could use as a key here (over an index?) This may be a good change to address some technical debt - https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
Note - we do have to be careful we do not break tests